docs/docs/apis/userinterface.md

```diff
@@ -57,5 +67,10 @@ dismissKeyboard(): void;
 // app is running
 contentSizeMultiplierChangedEvent: SubscribableEvent<
     (multiplier: number) => void>();
+
+// Triggered when the max content size multiplier changes while
+// the app is running.
+maxContentSizeMultiplierChangedEvent: SubscribableEvent<
```

Comment:
We don't need this event. The max content size multiplier is under the control of the app, so it can track this change itself. The reason I added an event for contentSizeMultiplierChanged is that it's under control of the OS (and ultimately the user), so the app needs to be informed when it changes.